src/shapes/path.cpp

```diff
@@ -70,8 +70,8 @@ void Path::buildPath(CommandPath& commandPath) const {
     } else {
         startIsCubic = prevIsCubic = false;
         auto point = *firstPoint->as<StraightVertex>();
-
-        if (auto radius = point.radius(); radius > 0.0f) {
```

Comment:
I like that this scoped "radius" to the if. Is there a reason it was removed?

Reply:
Is there somewhere we don't have C++17?

Reply:
Oops, some clarity. It was flagged as a "C++17 extension" -- when building w/ FFI for Flutter.